src/Mapbender/CoreBundle/Resources/public/mapbender.geosource.js

```diff
@@ -243,12 +246,12 @@ Mapbender.Geo.SourceHandler = Class({
                     if (layer.children[i].options.id.toString() === offsetLayer.options.id.toString()) {
                         options.found = true;
                         if (options.cut_with) {
-                            var lays = layer.children.splice(i, layer.children.length - i);
+                            var lays = layer.children.slice().splice(i, layer.children.length - i);
```

Comment:
Maybe use a better name for the variable lays. What does lays mean?

Reply:
Whole section cleaned up in [
af77e22](https://github.com/mapbender/mapbender/pull/852/commits/af77e2272abda2a742eaa7bef1839b3c04f25c71).